### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ashe546**

### CODE REVIEW

**Comment by Ashe546** [comment]
-created a component folder with the calculator.js file
- created the layout to the calculator
- Added some styling

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+import './App.css';
+
+export default function App() {
```
Comment (ShadyShawkat) [inline src/App.jsx]: - [ ] Kindly make sure that all of your components are class components, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=(All%20components%20should%20be%20class%2Dbased)) requirement.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Ashe546, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
Hi @Ashe546, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters Ok
- Good PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**